src/modules/didRegistry/didRegistry.service.ts

```diff
@@ -102,6 +104,31 @@ export class DidRegistry {
     };
   }
 
+  /**
+   * @description get claims from User's DID document
+   * @returns list of claims
+   */
+  async getDidClaims({ did = this._signerService.did }: { did?: string } | undefined = {}) {
```

Comment:
Since this actually returns service endpoints I suggest rename it to `readServices`. We can also add method to return actual offchain claims. I think for this we can extend `ClaimsService.getClaims...` with `registrationTypes` parameter. From my perspective ther is no need to wrap `did` in object

Reply:
My thoughts are:
- I definitely agree wth the rename to `readServices`.
- I don't think we should add a method to return actual offchain claims at this point as this will not be possible once we move to a presentation flow.

Reply:
+1